docs/integrations/sources/quickbooks.md

```diff
@@ -1,12 +1,76 @@
 # QuickBooks
 
-## Overview
+This page contains the setup guide and reference information for the QuickBooks Source connector.
 
-The QuickBooks source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+## Prerequisites
 
-This source wraps the [Singer QuickBooks Tap](https://github.com/singer-io/tap-quickbooks).
+- Account at [Intuit Developer account](https://developer.intuit.com/app/developer/qbo/docs/get-started)
+- Start date
```

Comment:
this shouldn't be in the pre-requisites section. Only things which need upfront preparation should be here e.g: if I need to go obtain an API key _before_ I start setting up the connector.

Reply:
fixed